extensions/project-descriptor.md

```diff
@@ -113,12 +154,12 @@ Any files that are excluded (either via `include` or `exclude`) MUST BE excluded
 
 If both `exclude` and `include` are defined, the build process MUST result in an error.
 
-### `[[build.buildpacks]]`
+#### `io.buildpacks.build.buildpacks` (optional)
 
-The build table MAY contain an array of buildpacks. The schema for this table is:
+This table MAY contain an array of buildpacks. The schema for this table is:
 
 ```toml
-[[build.buildpacks]]
+[[io.buildpacks.build.buildpacks]]
 id = "<buildpack ID (optional)>"
 version = "<buildpack version (optional default=latest)>"
 uri = "<url or path to the buildpack (optional default=urn:buildpack:<id>)"
```

Comment:
```suggestion
uri = "<url or path to the buildpack (optional default=urn:buildpack:<id>@<version>)"
```

Reply:
Nevermind, just read the line below this. Given `id` and `uri` are mutually exclusive defining the `uri` default in terms of `id` makes no sense...